src/js/helpers/searchWorker.js

```diff
@@ -1,23 +1,26 @@
 export function searchWorker(currentResults, searchString) {
+  if (typeof searchString === 'number') {
+    searchString = searchString.toString();
+  }
 
```

Comment:
the only thing I added. the rest was js prettier to try to trigger builds .

Reply:
👍